docs/settings.md

```diff
@@ -38,8 +38,11 @@ If Uvicorn _cannot_ load [watchfiles](https://pypi.org/project/watchfiles/) at r
 
 ### Reloading with watchfiles
 
-For more nuanced control over which file modifications trigger reloads, install `uvicorn[standard]`, which includes watchfiles as a dependency. Alternatively, install [watchfiles](https://pypi.org/project/watchfiles/) where Uvicorn can see it. This will enable the following options (which are otherwise ignored).
+For more nuanced control over which file modifications trigger reloads, install `uvicorn[standard]`, which includes watchfiles as a dependency. Alternatively, install [watchfiles](https://pypi.org/project/watchfiles/) where Uvicorn can see it.
 
+Note that if you use Uvicorn through [WSL](https://en.wikipedia.org/wiki/Windows_Subsystem_for_Linux), for example when running a Docker container on Windows, you might have to set the environment variable `WATCHFILES_FORCE_POLLING`, for file changes to trigger a reload. See [watchfiles documentation](https://watchfiles.helpmanual.io/api/watch/) for further details.
+
+Using Uvicorn with watchfiles will enable the following options (which are otherwise ignored).
```

Comment:
```suggestion
Using Uvicorn with watchfiles will enable the following options (which are otherwise ignored).

```
😅 
<img width="809" alt="image" src="https://user-images.githubusercontent.com/7353520/190204269-1d2b8b8f-3699-4961-bb80-b96452a25632.png">